setup.py

```diff
@@ -6,15 +6,15 @@
 
 name = "ripe.atlas.sagan"
 install_requires = [
+    "IPy",
```

Comment:
This isn't part of this issue is it?  If not, it should probably be in a separate commit, if only to keep future developers from conflating IPy with the addition of cryptography.  The same goes for the documentation references to IPy above.